lib/cosmos/script/api_shared.rb

```diff
@@ -23,12 +23,12 @@ module ApiShared
     #   get_packet(id, <true or false to block>)
     def get_packet(id, non_block = false)
       packet = nil
-      buffer, target_name, packet_name, received_time, received_count = get_packet_data(id, non_block)
+      buffer, target_name, packet_name, rx_sec, rx_usec, rx_count = get_packet_data(id, non_block)
       if buffer
         packet = System.telemetry.packet(target_name, packet_name).clone
         packet.buffer = buffer
-        packet.received_time = received_time
-        packet.received_count = received_count
+        packet.received_time = Time.at(rx_sec, rx_usec).sys
+        packet.received_count = rx_count
       end
       packet
     end
```

Comment:
Should probably just remove this from shared and redefine in both places.   Making the change to get_packet_data at the script level is breaking backwards compat.   Alternatively. could check the number of fields returned by get_packet_data.